specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/mfe.json

```diff
@@ -14232,6 +14226,22 @@
       "x-ms-discriminator-value": "Command",
       "additionalProperties": false
     },
+    "ComponentConfiguration": {
+      "description": "Used for sweep over component",
+      "type": "object",
+      "properties": {
+        "settings": {
```

Comment:
Should this be called pipeline_settings? As defined in the spec: https://github.com/Azure/azureml_run_specification/pull/916

Reply:
Fixed